electron/src/machines/aquapath/aquapath1/Aquapath1ControlPage.tsx

```diff
@@ -140,19 +92,86 @@ export function Aquapath1ControlPage() {
+                    Power: {front_power.current?.value} W<br />
+                    Total: {front_total_energy.current?.value} kWh
+                  </Badge>
+                )}
+              {frontCurrentTemperature > frontCoolingBoundary &&
+                frontTargetFlow &&
+                (state?.flow_states.front.flow ?? 0) && (
+                  <Badge variant="secondary">Cooling</Badge>
+                )}
+            </div>
+
+            <div className="flex flex-row">
+              <TimeSeriesValueNumeric
+                label="Revolution Speed"
+                unit="%"
+                timeseries={front_revolutions}
+                renderValue={(value) => value.toFixed(1)}
+              />
+            </div>
+
+            <div className="flex flex-row">
+              <Label label="Pump">
+                <SelectionGroup<"On" | "Off">
+                  value={frontTargetFlow ? "On" : "Off"}
+                  orientation="vertical"
+                  className="grid h-full grid-cols-2 gap-2"
+                  options={{
+                    Off: {
+                      children: "Off",
+                      icon: "lu:CirclePause",
+                      isActiveClassName: "bg-green-600",
+                      className: "h-full",
+                    },
+                    On: {
+                      children: "On",
+                      icon: "lu:CirclePlay",
+                      isActiveClassName: "bg-green-600",
+                      className: "h-full",
+                    },
+                  }}
+                  onChange={(value) => {
+                    setFrontFlow(value == "On");
+                  }}
+                />
+              </Label>
+            </div>
+          </div>
+        </ControlCard>
+
+        <ControlCard title="Reservoir 2">
+          <div className="grid grid-rows-4 gap-4">
+            <div className="flex flex-row">
+              <TimeSeriesValueNumeric
+                label="Flow"
+                unit="l/min"
+                timeseries={back_flow}
+                renderValue={(value) => value.toFixed(1)}
+              />
             </div>
 
-            <div className="flex flex-col gap-8">
+            <div className="flex flex-row">
               <TimeSeriesValueNumeric
-                label="Reservoir 2"
+                label="Temperature"
                 unit="C"
                 timeseries={back_temperature}
                 renderValue={(value) => value.toFixed(1)}
               />
-              <Label label="Set Reservoir 2 Target Temperature">
+            </div>
+
+            <div className="flex flex-row gap-4">
+              <Label label="Set Target Temperature">
                 <EditValue
-                  title="Set Reservoir 2 Target Temperature"
+                  title="Set Target Temperature"
+                  min={0}
                   value={backTargetTemperature}
+                  max={80}
```

Comment:
Out of curiosity: does your backend receive and clamp those values to avoid tampering?